src/components/submission-common.test.js

```diff
@@ -85,4 +85,20 @@ describe('resolveWdsUrl', () => {
     ]
     expect(resolveWdsUrl(mockAppList)).toBe(firstWdsProxyUrl)
   })
+
+  it.each(
+    [
+      { appStatus: 'RUNNING', expectedUrl: mockWdsProxyUrl },
+      { appStatus: 'PROVISIONING', expectedUrl: '' },
+      { appStatus: 'STOPPED', expectedUrl: '' },
+      { appStatus: 'STOPPING', expectedUrl: '' }
+    ]
+  )('gives precedence to the WDS appType over the CROMWELL appType', ({ appStatus, expectedUrl }) => {
+    const uuid = '123e4567-e89b-12d3-a456-426614174000'
+    const testHealthyAppProxyUrlResponse: Array<Object> = [
```

Comment:
The type definition `Array<Object>` is needed for Typescript files only. We can remove this here.

Reply:
Thank you for catching this!